warehouse/templates/packaging/detail.html

```diff
@@ -147,14 +149,39 @@ <h1 class="package-header__name">
       </h1>
 
       {% if files %}
-      <p class="package-header__pip-instructions">
-        <span id="pip-command">{{ pip_command(release, request.matched_route.name, testPyPI) }}</span>
-        <button type="button" class="copy-tooltip copy-tooltip-s" data-clipboard-target="#pip-command" data-tooltip-label="{% trans %}Copy to clipboard{% endtrans %}">
-          <i class="fa fa-copy" aria-hidden="true"></i>
-          <span class="sr-only">{% trans %}Copy PIP instructions{% endtrans %}</span>
-        </button>
-      </p>
-      {% endif %}
+      <div
+        data-controller="package-manager-instructions"
+        data-package-manager-instructions-package-name-value="{{ release.project.name }}"
+        data-package-manager-instructions-project-version-value="{{ project_version }}"
+        data-package-manager-instructions-index-url-value="{{ index_url }}"
+      >
+        <p class="package-header__pip-instructions">
+          <span id="pip-command" data-package-manager-instructions-target="command">{{ pip_command(release, request.matched_route.name, testPyPI) }}</span>
+          <button type="button" class="copy-tooltip copy-tooltip-s" data-clipboard-target="#pip-command" data-tooltip-label="{% trans %}Copy to clipboard{% endtrans %}">
+            <i class="fa fa-copy" aria-hidden="true"></i>
+              <!-- todo: rename this to installation instructions? -->
+              <span class="sr-only">{% trans %}Copy PIP instructions{% endtrans %}</span>
```

Comment:
Yes please - since `sr-*` prefixes are for screen readers. If you can shim in the package manager name here, that'd be cool too!